pennylane/templates/subroutines/qchem/uccsd.py

```diff
@@ -220,8 +234,13 @@ def __init__(
                 f"Weights tensor must be of shape {(n_repeats,) + expected_shape}; got {shape}."
             )
 
+        if init_state is None:
+            raise ValueError("Requires `init_state` to be provided.")
```

Comment:
I guess we have to do this to retain the function signature. But we can at least bump it to the top so that we can do an early exit.